addons/event/views/event_views.xml

```diff
@@ -182,6 +182,30 @@
            <field name="view_mode">tree,kanban,form,calendar,graph</field>
         </record>
 
+        <record id="event_registration_action_confirm" model="ir.actions.server">
```

Comment:
To be able to use them in the action menu in the tree view :)

![_tmp_screenshot](https://user-images.githubusercontent.com/54316226/72433676-2d234980-379a-11ea-890a-1a697d0f008d.png)

Reply:
Why not enabling a multi edit on state ?

Reply:
@tde-banana-odoo where ? :O

Reply:
Multi edit on tree view of registrations, allowing to update state of registrations on all selected registrations at once. See multi_edit="1" in views.

In that case you will have to update the various do_draft / confirm_registration / button_reg_close to have their behavior supported in write instead of specific actions. For example confirm_registration perform a write on state with state = open; in write we have to intercept a write on state to open in order to call the mail / sms schedulers.

Reply:
Also pay attention to limit multi edit to fields without onchange, as those are not trigerred in the list view in multi edit.